app/main.py

```diff
@@ -1,8 +1,36 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.wife = None
+        self.husband = None
+        self.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people_data: list) -> list:
+    people = []
+    for person_dict in people_data:
+        name = person_dict.get("name")
+        age = person_dict.get("age")
+        person = Person(name, age)
+
+        if "wife" in person_dict and person_dict["wife"] is not None:
```

Comment:
Simplify if statement and just use: person_dict.get("wife") is not None
The same with husband